test/test-fork.c

```diff
@@ -346,6 +346,48 @@ TEST_IMPL(fork_signal_to_child_closed) {
   return 0;
 }
 
+static void fork_signal_cb(uv_signal_t* h, int s) {
+  fork_signal_cb_called = s;
+}
+static void empty_close_cb(uv_handle_t* h){}
+
+TEST_IMPL(fork_close_signal_in_child) {
+  uv_loop_t loop;
+  uv_signal_t signal_handle;
+  pid_t child_pid;
+
+  uv_loop_init(&loop);
+  uv_signal_init(&loop, &signal_handle);
+  uv_signal_start(&signal_handle, &fork_signal_cb, SIGHUP);
+
+  fork_signal_cb_called = 0;
+
```

Comment:
```suggestion
```
Should not be necessary to initialize since it lives in static storage, I think?

Reply:
I am note sure if it is right. As I know all the tests run in one process and can influence each other. This variable is used in other tests so it should be initialized again. All other isssues are fixed.

Reply:
Each test runs in its own process. There's a test runner process that spawns a new process for each test.

Reply:
Ok, it makes sense. I copied this line from other tests. Would it be better to remove it from all tests that use 'fork_signal_cb_called' variable?

Reply:
The line has been removed.